cypher/cypher-docs/src/docs/dev/ql/administration/constraints/index.asciidoc

```diff
@@ -103,14 +103,14 @@ Index provider and configuration for the backing index can be specified using th
 ----
 CREATE CONSTRAINT [constraint_name] [IF NOT EXISTS]
 ON (n:LabelName) 
-ASSERT EXISTS (n.propertyName)
+ASSERT n.propertyName IS NOT NULL
 ----
 | [enterprise-edition]#Create a node property existence constraint.#
 | [source, cypher]
 ----
 CREATE CONSTRAINT [constraint_name] [IF NOT EXISTS]
 ON ()-[R:RELATIONSHIP_TYPE]-() 
-ASSERT EXISTS (R.propertyName)
+ASSERT R.propertyName IS NOT NULL
```

Comment:
I know this already was this way,but to follow Cypher naming rules, the relationship variable should be a small `r`

Reply:
If I have other comments to fix I'll fix this as well, otherwise I'll skip it since it was this way before :)